sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceClientBuilder.java

```diff
@@ -114,7 +114,7 @@ public BlobServiceAsyncClient buildAsyncClient() {
             policies.add(new SASTokenCredentialPolicy(sasTokenCredential));
         }
         HttpPolicyProviders.addBeforeRetryPolicies(policies);
-        policies.add(new RequestRetryPolicy(retryOptions));
+       // policies.add(new RequestRetryPolicy(retryOptions));
```

Comment:
Why did we comment out this line of code? This is a critical line of code.

Reply:
Just for quick expose the error msg. Will uncomment. Good catch.